repo/packages/B/beta-tensorflow/0/config.json

```diff
@@ -0,0 +1,329 @@
+{
+  "type": "object",
+  "properties": {
+    "service": {
+      "type": "object",
+      "description": "DC/OS service configuration properties",
+      "properties": {
+        "name": {
+          "title": "Service name",
+          "description": "The name of the service instance",
+          "type": "string",
+          "default": "tensorflow"
+        },
+        "user": {
+          "title": "User",
+          "description": "The user that the service will run as.",
+          "type": "string",
+          "default": "root"
```

Comment:
Can this be nobody?

Reply:
It probably should, but I don't want to make this change now. I will work that into the next round.